src/ch09-03-to-panic-or-not-to-panic.md

```diff
@@ -1,20 +1,20 @@
 ## 要 panic! 還是不要 panic!
 
-所以你該如何決定何時要呼叫 panic! 還是要回傳 Result 呢？當程式碼恐慌時，就沒有任何回復的方式。你可以在任何錯誤場合呼叫 `panic!`，無論是可能或不可能復原的情況。不過這樣你就等於替呼叫你的程式碼的呼叫者做出決定，讓情況變成無法復原的錯誤了。當你選擇回傳 `Result` 數值，你將決定權交給呼叫者的程式碼。呼叫者可能會選擇符合當下場合的方式嘗試復原錯誤，或者它可以選擇 Err 內的數值是不可回復的，所以它就呼叫 panic! 讓你原本可回復的錯誤轉成不可回復。因此，當你定義可能失敗的函式時預設回傳 Result 是不錯的選擇。
+所以你該如何決定何時要呼叫 `panic!` 還是要回傳 Result 呢？當程式碼恐慌時，就沒有任何回復的方式。你可以在任何錯誤場合呼叫 `panic!`，無論是可能或不可能復原的情況。不過這樣你就等於替呼叫你的程式碼的呼叫者做出決定，讓情況變成無法復原的錯誤了。當你選擇回傳 `Result` 數值，你將決定權交給呼叫者的程式碼。呼叫者可能會選擇符合當下場合的方式嘗試復原錯誤，或者它可以選擇 `Err` 內的數值是不可回復的，所以它就呼叫 `panic!` 讓你原本可回復的錯誤轉成不可回復。因此，當你定義可能失敗的函式時預設回傳 `Result` 是不錯的選擇。
```

Comment:
Result also needs backtick?

```suggestion
所以你該如何決定何時要呼叫 `panic!` 還是要回傳 `Result` 呢？當程式碼恐慌時，就沒有任何回復的方式。你可以在任何錯誤場合呼叫 `panic!`，無論是可能或不可能復原的情況。不過這樣你就等於替呼叫你的程式碼的呼叫者做出決定，讓情況變成無法復原的錯誤了。當你選擇回傳 `Result` 數值，你將決定權交給呼叫者的程式碼。呼叫者可能會選擇符合當下場合的方式嘗試復原錯誤，或者它可以選擇 `Err` 內的數值是不可回復的，所以它就呼叫 `panic!` 讓你原本可回復的錯誤轉成不可回復。因此，當你定義可能失敗的函式時預設回傳 `Result` 是不錯的選擇。
```

Reply:
Yes, I guess. Refer to the original version [ch09-03](https://github.com/rust-lang/book/blob/master/src/ch09-03-to-panic-or-not-to-panic.md). All `Result`s are backquoted.

```markdown
So how do you decide when you should call `panic!` and when you should return
`Result`? When code panics, there’s no way to recover. You could call `panic!`
for any error situation, whether there’s a possible way to recover or not, but
then you’re making the decision on behalf of the code calling your code that a
situation is unrecoverable. When you choose to return a `Result` value, you
give the calling code options rather than making the decision for it. The
calling code could choose to attempt to recover in a way that’s appropriate for
its situation, or it could decide that an `Err` value in this case is
unrecoverable, so it can call `panic!` and turn your recoverable error into an
unrecoverable one. Therefore, returning `Result` is a good default choice when
you’re defining a function that might fail.
```